pkg/resource/deploy/plan.go

```diff
@@ -53,6 +53,7 @@ type Options struct {
 	RefreshOnly       bool           // whether or not to exit after refreshing.
 	RefreshTargets    []resource.URN // The specific resources to refresh during a refresh op.
 	DestroyTargets    []resource.URN // Specific resources to destroy.
+	UpdateTargets     []resource.URN // Specific resources to update.
```

Comment:
@pgavlin we could just collapse this into one 'targets' array.  it's not like multiple of them can be set at the same time.